app/views/planning_applications/index.html.erb

```diff
@@ -2,68 +2,74 @@
-      </a>
-    </div>
-    <div class="status-panel<% if filter_enabled_uniquely?(application_type: 'prior_approval') %> status-panel--highlighted<% end %>" id="not-started">
-      <a class="status-filter-link" href="<%= planning_applications_path(filter_enabled_uniquely?(application_type: "prior_approval") ? {} : { application_type: [:prior_approval] }) %>">
-        <h3 class="govuk-heading-s"><%= t ".prior_approvals.title" %></h3>
-        <p class="govuk-heading-m"><%= t ".prior_approvals", count: @search.current_planning_applications.prior_approvals.not_started.count %></p>
-      </a>
+  <div class="status-bar-container">
+    <h2 class="govuk-heading-m">Status update:</h2>
+    <div class="status-bar">
+      <div class="status-panel<% if filter_enabled_uniquely?(status: 'to_be_reviewed') %> status-panel--highlighted<% end %>" id="reviewer-requests">
+        <a class="status-filter-link" href="<%= planning_applications_path(filter_enabled_uniquely?(status: "to_be_reviewed") ? {} : { status: [:to_be_reviewed] }) %>">
+          <h3 class="govuk-heading-s"><%= t ".reviewer_requests.title" %></h3>
+          <p class="govuk-heading-m"><%= t ".reviewer_requests", count: @search.current_planning_applications.to_be_reviewed.count %></p>
+        </a>
+      </div>
+      <div class="status-panel<% if filter_enabled_uniquely?(application_type: 'prior_approval') %> status-panel--highlighted<% end %>" id="not-started">
+        <a class="status-filter-link" href="<%= planning_applications_path(filter_enabled_uniquely?(application_type: "prior_approval") ? {} : { application_type: [:prior_approval] }) %>">
+          <h3 class="govuk-heading-s"><%= t ".prior_approvals.title" %></h3>
+          <p class="govuk-heading-m"><%= t ".prior_approvals", count: @search.current_planning_applications.prior_approvals.not_started.count %></p>
+        </a>
+      </div>
     </div>
   </div>
-</div>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <div class="govuk-tabs" data-module="govuk-tabs">
-      <h2 class="govuk-tabs__title">
-        Contents
-      </h2>
-      <%= render "planning_application_tabs" %>
-      <%= render(
-        PlanningApplications::PanelsComponent.new(
-          planning_applications: @planning_applications,
-          search: @search,
-          local_authority: current_local_authority
-        )
-      ) %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <div class="govuk-tabs" data-module="govuk-tabs">
+        <h2 class="govuk-tabs__title">
+          Contents
+        </h2>
+        <%= render "planning_application_tabs" %>
+        <%= render(
+          PlanningApplications::PanelsComponent.new(
+            planning_applications: @planning_applications,
+            search: @search,
+            local_authority: current_local_authority
+          )
+        ) %>
+      </div>
     </div>
   </div>
-</div>
```

Comment:
How about moving this logic on a `before_action` in the controller similar to [this](https://github.com/unboxed/bops/blob/main/app/controllers/application_controller.rb#L73-L77) ?

Reply:
I thought about this, but the issue is that this would prevent the Administrator from logging in because we can't detect their role until after they have logged in. So it would have to apply only after the user has logged in. And in that case they need to be able to log out, so they need to see the nav bar or else the sessions get messed up. I kind of get that it's technically possible, but not sure how to implement it.

Reply:
you could expand and add a specific method [here](https://github.com/unboxed/bops/blob/main/app/controllers/users/sessions_controller.rb#L13)